drivers/net/Kconfig

```diff
@@ -392,6 +392,18 @@ menuconfig NET_W5500
 		References:
 		W5500 Datasheet, Version 1.0.9, 2013 WIZnet Co., Ltd.
 
+if NET_W5500
+
+config NET_W5500_NINTERFACES
+	int "Number of physical W5500 devices"
+	default 1
+	range 1 1
```

Comment:
Do we expect to extend maximum value (otherwise config option does not make sense)?

Reply:
In this PR, that block was just moved inside the file, while adding options related to the switch. It had somehow ended up in the middle of ksz configs.